addons/website_slides/models/slide_slide.py

```diff
@@ -126,16 +130,18 @@ class Slide(models.Model):
         ('infographic', 'Infographic'),
         ('presentation', 'Presentation'),
         ('document', 'Document'),
+        ('webpage', 'Web Page'),
         ('video', 'Video')],
         string='Type', required=True,
-        default='document',
+        default='document', readonly=True,
         help="The document type will be set automatically based on the document URL and properties (e.g. height and width for presentation and document).")
     index_content = fields.Text('Transcript')
```

Comment:
While we are at it, shouldn't index content be computed as plaintext version of html content for webpage slides ?

Reply:
Can not be a computed field, as if the slide is a PDF, it will contains the transcript of the file. Moreover, how to handle this is, if html_content is a translatable field ? 0_O